### DISCUSSION TITLE
WalletConnect QRCODE didn't work

### DISCUSSION THREAD

**Discussion by LuHank** [WalletConnect/walletconnect-monorepo#4274] [Developer Support]
I code an Dapp to let user choose Wallet.
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/46447100/c70da9d8-90fe-4892-9624-e3b89f5e7fe3)
When user choose WalletConnect wallet, it will generate QRCODE page to let user scan.
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/46447100/5abfe273-3cf2-41da-ac16-fa8af1f5afc0)
Every time user first scan QRCODE by MetaMask , MetaMask wallet will show connect successfully. But when user continue to next step on Dapp website, and same process to let user choose WalletConnect wallet and generate QRCODE. This time user scan QRCODE, the MetaMask didn't show connect message.
You must wait 2 to 3 minutes, the QRCODE will refresh, then you can scan successfully.
I found it didn't show auth as the below immediately at second time.
wss://relay.walletconnect.com/?auth=~
Who can help me how to solve the problem ?
Thanks.

**Comment by glitch-txs** [comment]
Hi, could you share a minimal reproducible example and steps to reproduce the issue? Thanks!
